src/aap_eda/api/serializers/event_stream.py

```diff
@@ -41,6 +42,22 @@ class EventStreamInSerializer(serializers.ModelSerializer):
         ],
         error_messages={"null": "EdaCredential is needed"},
     )
+    uuid = serializers.UUIDField(required=False, allow_null=True)
+
+    def validate_uuid(self, value):
```

Comment:
@hsong-rh Wont Django's unique=True and we have a default set is this code still needed

Reply:
@mkanoor Use this validation function; we can handle exceptions better. Otherwise, we need to change in views to catch errors if they happen.